tests/generate_plan_tests.rs

```diff
@@ -341,12 +342,11 @@ fn test_custom_rust_version() -> Result<()> {
 #[test]
 fn test_rust_rocket() -> Result<()> {
     let plan = simple_gen_plan("./examples/rust-rocket");
-    assert_eq!(
-        plan.build.unwrap().cmds,
-        Some(vec![
-            "cargo build --release --target x86_64-unknown-linux-musl".to_string()
-        ])
+    let cmd = format!(
+        "cargo build --release --target {}-unknown-linux-musl",
+        consts::ARCH
```

Comment:
IMO, it would be better to explicitly import `ARCH` from the use that was added.

Reply:
done